GOVERNANCE.md

```diff
@@ -0,0 +1,129 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, led, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision-making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+**Maintainers** are the people who contribute, review, guide, and collectively
+make decisions about the direction and scope of the project (see:
+[Decision Making](#decision-making)). Maintainers are elected by a
+[voting process](#adding-and-removing-maintainers).
+
+A typical Maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+  being a role model for others.
+- Writing patches &mdash; a lot of patches, especially "glue code" or "grunt
+  work" or general "housekeeping"; fixing bugs, ensuring documentation is always
+  high quality, consistent UX design, improving processes, making judgments on
+  dependencies, handling security vulnerabilities, and so on and so forth.
+- Reviewing code submitted by others &mdash; with an eye to maintainability,
+  performance, code quality, and "style" (fitting in with the project).
+- Participating in design discussions, especially with regards to architecture
+  or long-term vision.
```

Comment:
I would explicitly add something about making sure the community remains a welcoming place to users and contributors

Reply:
Yeah, thanks for this comment, I totally agree - all maintainers should be doing this, or at the *very least* should not be having the opposite effect. Would love to have that written down.